servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/FullDuplexAndSequentialModeTest.java

```diff
@@ -90,10 +90,11 @@ private static InputStream payload() {
         return new BufferedInputStream(new ByteArrayInputStream(array));
     }
 
-    private static Future<StreamingHttpResponse> stallingSendRequest(StreamingHttpConnection connection,
+    private Future<StreamingHttpResponse> stallingSendRequest(StreamingHttpConnection connection,
                                                                      CountDownLatch continueRequest,
                                                                      InputStream payload) {
-        return connection.request(connection.post(SVC_ECHO).payloadBody(fromInputStream(payload, CHUNK_SIZE)
+        return connection.request(connection.post(SVC_ECHO).payloadBody(fromInputStream(payload, 1)
```

Comment:
Why did the CHUNK_SIZE change from 1024 to 1 here?

Reply:
Because if it stays what it was the `assertThrows(..)` doesn't happen. I suspect we end up racing the cancellation in the separate thread.

Reply:
Now, knowing the reason, I'm agreed that smaller chunk makes it likely to happen we end up in a deadlock. For the original purpose of the test, it should not matter though. So, I don't mind if we change the size here or not.